components/p3a/rotation_scheduler_unittest.cc

```diff
@@ -59,7 +59,10 @@ TEST_F(P3ARotationSchedulerTest, JsonRotation) {
 }
 
 // TODO(djandries): find a way to get this test to run reliably on macOS
-#if !BUILDFLAG(IS_MAC)
+#if BUILDFLAG(IS_MAC)
```

Comment:
there is a better pattern, e.g. https://github.com/brave/brave-core/blob/master/browser/farbling/brave_font_whitelist_browsertest.cc#L62

this way it's easier to understand that there are nuances, when looking at the test itself